transiXNOR/plotting.py

```diff
@@ -360,14 +360,11 @@ def gate_charges(Vds, Vbg, Vtg, er, model_path):
 					np.abs(gridx[Nx / 2] - gridx[Nx - 1]) * 1e-9)  # C/m^2
 				Q_top.append(Q_top_x)
 				Q_bottom.append(Q_bottom_x)
-		np.save(model_path + '/charges/TG_Q_' + voltage, Q_top)  # save top gate charges
-		np.save(model_path + '/charges/BG_Q_' + voltage, Q_bottom)  # save bottom gate charges
+		# np.save(model_path + '/charges/TG_Q_' + voltage, Q_top)  # save top gate charges
+		# np.save(model_path + '/charges/BG_Q_' + voltage, Q_bottom)  # save bottom gate charges
 		Q_top_total = np.sum(Q_top)  # C/m^2
 		Q_bottom_total = np.sum(Q_bottom)  # C/m^2
-		if (V != 0.0):
-			C = (Q_top_total + Q_bottom_total) / (Vtg + Vbg)
-		else:
-			C = 0.175 # Because all calcualted C are close to 0.175
+		C = er*eps0/(1e-9)
```

Comment:
C should Cox and Csemi in series. If you use er_que as er, C = er*eps0/(np.abs(gridx[Nx / 2] - gridx[Nx - 1]) * 1e-9)